core/store/src/main/java/uk/gov/gchq/gaffer/store/Store.java

```diff
@@ -385,14 +385,24 @@ public <O> O execute(final Output<O> operation, final Context context) throws Op
         return execute(OperationChain.wrap(operation), context);
     }
 
+    private boolean isJobOperation(Class<?> operationClass) {
```

Comment:
Not sure this is solving the issue, I think you want to basically remove all the job tracker stuff from the `execute` method so that it only updates things submitted to the `executeJob` method

Reply:
fixed